AndorsTrail/res/raw/questlist_stoutford.json

```diff
@@ -6,31 +6,31 @@
         "stages":[
             {
                 "progress":10,
-                "logText":"The priest in Stoutford looks scared of me and of what I did, but we never met before. Can this have something to do with Andor ?"
+                "logText":"The priest in Stoutford looks scared of me and of what I did, but we never met before. Can this have something to do with Andor?"
             },
             {
                 "progress":20,
-                "logText":"The priest's acolyte, Yolgen, told me that Andor was here a while ago, and since then, scary noises resonate in the church, seemingly coming from the underground."
+                "logText":"The priest's acolyte, Yolgen, told me that Andor was here a while ago, and since then, scary noises resonate in the church, seemingly coming from underground."
             },
             {
                 "progress":25,
                 "logText":"A member of the Thieve's Guild in the tavern of Stoutford told me that the tavern owner dealt with Andor."
             },
             {
                 "progress":30,
-                "logText":"Glasforn, the owner of the tavern of Stoutford, admits to knowing Andor, but evades all my questions by boasting about his beds' quality."
+                "logText":"Glasforn, the owner of the tavern of Stoutford, admits to knowing Andor, but evades all my questions by boasting about his bed's quality."
```

Comment:
I fixed the typos you found, and the ones I found, and two more that neither of us found (until now).

Could you explain what you mean regarding the negated requirements. I sort of remember creating them, for reasons I'm sure seemed logical at the time. Is there a specific problem with them, or it's just that they are unusual?